yarn.lock

```diff
@@ -8066,6 +8066,11 @@ graceful-fs@^4.1.11, graceful-fs@^4.1.15, graceful-fs@^4.1.2, graceful-fs@^4.1.6
   resolved "https://registry.yarnpkg.com/graceful-fs/-/graceful-fs-4.2.9.tgz#041b05df45755e587a24942279b9d113146e1c96"
   integrity sha512-NtNxqUcXgpW2iMrfqSfR73Glt39K+BLwWsPs94yR63v45T0Wbej7eRmL5cWfwEgqXnmjQp3zaJTshdRW/qC2ZQ==
 
+grapheme-splitter@^1.0.4:
+  version "1.0.4"
+  resolved "https://registry.yarnpkg.com/grapheme-splitter/-/grapheme-splitter-1.0.4.tgz#9cf3a665c6247479896834af35cf1dbb4400767e"
+  integrity sha512-bzh50DW9kTPM00T8y4o8vQg89Di9oLJVLW/KaOGIXJWP/iqCN6WKYkbNOF04vFLJhwcpYUh9ydh/+5vpOqV4YQ==
```

Comment:
Funny that ESLint added this dependency in a version after what we're currently using. https://github.com/eslint/eslint/blame/a682562458948f74a227be60a80e10e7a3753124/package.json#L80 -> https://github.com/eslint/eslint/pull/16154. `1.0.4` is the same version. So folks who are using a relatively new ESLint version will be deduped.

Reply:
I see... So what should we do?

Reply:
Oh sorry I didn't mean to imply anything change-request-like here. All is well!